### PR TITLE
add domain packet dissector, improve entity dissector

### PR DESCRIPTION
**Pull request by birarda**

### CODE REVIEW

**Comment by birarda** [comment]
Won't need QA - changes to Wireshark dissectors only

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/10396/HighFidelity-Beta-PR13320-f9350d7.apk

File: tools/dissectors/hf-entity.lua [hunk 1]
```diff
@@ -4,33 +4,94 @@ p_hf_entity = Proto("hf-entity", "HF Entity Protocol")
...
-  i = 0
+  local i = 0
 
   entity_subtree:add_le(f_entity_sequence_number, buf(i, 2))
   i = i + 2
 
-  entity_subtree:add_le(f_entity_timestamp, buf(i, 4))
-  i = i + 4
+  entity_subtree:add_le(f_entity_timestamp, buf(i, 8))
+  i = i + 8
```
Comment (huffman) [inline tools/dissectors/hf-entity.lua]: was this wrong before?

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/10396/HighFidelity-Beta-PR13320-f9350d7.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/10396/HighFidelity-Beta-Interface-PR13320-f9350d7.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/10396/HighFidelity-Beta-PR13320-f9350d7.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/10396/HighFidelity-Beta-Interface-PR13320-f9350d7.exe

File: tools/dissectors/hf-entity.lua (hunk 1 above)
Comment (birarda) [reply tools/dissectors/hf-entity.lua]: Yeah, it's 64-bit

File: tools/dissectors/hf-entity.lua [hunk 2]
```diff
@@ -4,33 +4,94 @@ p_hf_entity = Proto("hf-entity", "HF Entity Protocol")
...
 
-  -- figure out the number of bytes the octal code takes
-  local octal_code_bytes = buf(i, 1):le_uint()
-  entity_subtree:add_le(f_octal_code_bytes, buf(i, 1))
+  -- figure out the number of three bit sections in the octal code takes
```
Comment (huffman) [inline tools/dissectors/hf-entity.lua]: Remove "takes" at the end?

File: tools/dissectors/hf-entity.lua [hunk 3]
```diff
@@ -4,33 +4,94 @@ p_hf_entity = Proto("hf-entity", "HF Entity Protocol")
...
+  f_coded_update_delta, f_update_delta
 }
 
 function p_hf_entity.dissector(buf, pinfo, tree)
   pinfo.cols.protocol = p_hf_entity.name
 
   entity_subtree = tree:add(p_hf_entity, buf())
 
-  i = 0
+  local i = 0
```
Comment (huffman) [inline tools/dissectors/hf-entity.lua]: We use `i` here but `byte` in the domain packet dissector. Probably not worth making them consistent, but wanted to point it out.

Comment (birarda) [reply tools/dissectors/hf-entity.lua]: I changed the one in domain to match

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/10470/HighFidelity-Beta-PR13320-e82edb8.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/10470/HighFidelity-Beta-PR13320-e82edb8.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/10470/HighFidelity-Beta-Interface-PR13320-e82edb8.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/10470/HighFidelity-Beta-PR13320-e82edb8.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/10470/HighFidelity-Beta-Interface-PR13320-e82edb8.exe

**Pull request merged**